opencti-platform/opencti-front/src/private/components/settings/notifiers/NotifierEdition.tsx

```diff
@@ -126,19 +127,37 @@ const NotifierEdition: FunctionComponent<NotifierEditionComponentProps> = ({
     authorized_members: data?.authorized_members?.map(({ id, name }) => ({ value: id, label: name })) ?? [],
     notifier_connector_id: data?.notifier_connector ? { value: data.notifier_connector.id, label: data.notifier_connector.name } : undefined,
   };
-  const submitForm = (setSubmitting: FormikHelpers<NotifierEditionValues>['setSubmitting'], values: NotifierEditionValues, current: FormRef) => {
-    if (current?.validateForm()) {
-      setSubmitting(true);
-      const inputs = [
-        { key: 'name', value: [values.name] },
-        { key: 'description', value: [values.description] },
-        { key: 'authorized_members', value: values.authorized_members?.map(({ value }) => value) },
-        { key: 'notifier_connector_id', value: [values.notifier_connector_id?.value] },
-        { key: 'notifier_configuration', value: [JSON.stringify(current.state.formData)] },
-      ];
-      commitFieldPatch({ variables: { id: data?.id, input: inputs } });
-      setSubmitting(false);
-    }
+  const submitForm = (
+    setSubmitting: FormikHelpers<NotifierEditionValues>['setSubmitting'],
+    setErrors: FormikHelpers<NotifierEditionValues>['setErrors'],
+    values: NotifierEditionValues,
+    current: FormRef,
+  ) => {
+    notifierValidation(t)
+      .validate(values)
+      .then(() => {
```

Comment:
You shouldn't have its lines since the validator is declared on the formik side
validationSchema={notifierValidation(t)}
You need to disabled the button if the form is not valid

Reply:
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/139a095e-6cd9-4d5f-a0aa-996af6e7ee10)

Reply:
This is indeed the case for creation, but for editing has started to be implemented.
Example in ReportEditionOverview file.

Reply:
yes but the notifier editor is constructed like a creation form: there is a Cancel and a Save button. Whereas entities edition forms are constructed without such submission button (update is done at field changing).

Reply:
seen with Romu